### ISSUE TITLE
Editing cell background color

### ISSUE THREAD

**Issue by Romeo84095** [unvell/ReoGrid#262]
Hi! Could you pls advise, how can i change editing cell background color? (win forms app)

Here the screenshot

<a href='https://postimg.cc/PCPFfMsH' target='_blank'><img src='https://i.postimg.cc/PCPFfMsH/sample.jpg' border='0' alt='sample'/></a>

**Comment by Eireannanl** [comment]
I tried to reproduce that your problem but couldn't. Here's what I got.
![devenv_2019-07-27_02-09-32](https://user-images.githubusercontent.com/9781825/61985805-97d9a080-b013-11e9-88a0-01ec32c5aefd.png)

I used following code
```
var sheet = reoGridControl1.CurrentWorksheet;

var range = sheet.Ranges[0, 0, 5, 5];
range.Style.BackColor = SolidColor.Black;
range.Style.TextColor = SolidColor.White;

sheet.SetRangeBorders(range, BorderPositions.InsideAll,
new RangeBorderStyle
{
     Color = SolidColor.White,
     Style = BorderLineStyle.Solid
});
```

Hopefully this will solve your problem :)

**Comment by jingwood** [comment]
Use this can change the edit-box background. But edit-text will also be changed to black so this need to be fixed.

```csharp
grid.ControlStyle.SetColor(ControlAppearanceColors.GridBackground, Color.Black);
```

**Comment by jingwood** [comment]
This problem has been resolved since v3.0.0, please use the latest packages.
